### PR TITLE
http: do not crash if extra assets folder does not exist

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

**Comment by lebauce** [comment]
run skydive-selenium-tests

File: http/server.go [hunk 1]
```diff
@@ -268,6 +268,10 @@ func (s *Server) loadExtraAssets(folder, prefix string) {
 	files := []string{}
 
 	err := filepath.Walk(folder, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
```
Comment (masco-enovance) [inline http/server.go]: No return params for this function. look like we can't return the error here, just log the error and return nil is fine.

Comment (lebauce) [reply http/server.go]: There is already a line that logs this error line 280

Comment (masco-enovance) [reply http/server.go]: got it. Thanks!

**Pull request merged**